BackofficeBundle/Resources/views/BackOffice/Underscore/widgetVersion._tpl.twig

```diff
@@ -1,5 +1,6 @@
 <option class="version-select" value="<%= element.get('version') %>" <% if (element.get('version') == version) { %>selected="selected"<% } %>>
     <b class="page_status_title"><%= element.get('status').label %></b>
     <%= element.get('name') %>
-    ({{ 'open_orchestra_backoffice.element.version.display' | trans({}, 'messages', language) }} : #<%= element.get('version') %>)
+    ({{ 'open_orchestra_backoffice.element.version.display' | trans({}, 'messages', language) }} : #<%= element.get('version') %>, 
+     {{ 'open_orchestra_backoffice.element.version.updated' | trans({}, 'messages', language) }} : <%= element.get('updated_at') %>)
```

Comment:
@itkg-ngilain I'm displaying this 12/04/2016 17:10:14 for both languages (en and fr). dd/mm/yyyy hh:mm:ss